examples/nlp/language_modeling/tuning/megatron_gpt_peft_tuning.py

```diff
@@ -85,7 +85,21 @@ def _modify_config(gpt_cfg, cfg, add_cfg_to_tree=False):
         gpt_cfg.activations_checkpoint_granularity = cfg.model.get("activations_checkpoint_granularity", None)
         gpt_cfg.activations_checkpoint_num_layers = cfg.model.get("activations_checkpoint_num_layers", None)
         gpt_cfg.activations_checkpoint_method = cfg.model.get("activations_checkpoint_method", None)
+        gpt_cfg.activations_checkpoint_layers_per_pipeline = cfg.model.get(
```

Comment:
what does this do and why is this needed?

Reply:
We call `_reset_activation_checkpointing_args()` and `_restore_activation_checkpointing_args()` in validation and test step start ([code](https://github.com/NVIDIA/NeMo/blob/fix-gpt-trainer-test/nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py#L669)) which we need the config of `activations_checkpoint_layers_per_pipeline`.